### PR TITLE
Updates to the latest warden

### PR DESCRIPTION
**Pull request by rauhryan**

### CODE REVIEW

**Comment by discorick** [comment]
Is this just an update to the `minitest` gem? I don't see any other changes.

File: Gemfile.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 GIT
   remote: git://github.com/huboard/warden-github.git
-  revision: 459903681fa00820065348c27639b9f27db9fa55
+  revision: 3c3cf6943b01ff12cfe22a1ad1202ddcc4721e66
```
Comment (rauhryan) [inline Gemfile.lock]: ^ updates our warden-github gem to the lastest git sha on https://github.com/huboard/warden-github

**Comment by rauhryan** [comment]
> Is this just an update to the minitest gem? I don't see any other changes.

not sure why bundler wanted to update minitest, but the most important change is the sha change for the huboard/warden-github gem

File: Gemfile.lock (hunk 1 above)
Comment (discorick) [inline Gemfile.lock]: Oh interesting, just for my own learning here, does that mean when looking for the warden gem, it grabs that remote and that sha on every (gem) install ?

Comment (rauhryan) [inline Gemfile.lock]: Yep, we have a dependency on so custom code in the warden-github gem to figure out what scopes we have requested. 

The Gemfile.lock specifies that we want that exact sha. 

So, our heroku build will pull that new sha on the next deploy

**Pull request merged**